src/cr/cube/cubepart.py

```diff
@@ -806,8 +806,8 @@ def var_scale_means_row(self):
         return np.nansum(numerator, axis=0) / denominator
 
     @lazyproperty
-    def zscore(self):
-        return np.array([row.zscore for row in self._matrix.rows])
+    def zscores(self):
```

Comment:
I don't know if there's a higher level API that uses this, and describes what it is, but a docstring surely would be nice. Again, the comment from the commit message (https://github.com/Crunch-io/crunch-cube/pull/216/commits/128dbbc73e40ecda563c3ee185743804616b8b2c) seems valuable.

Reply:
Yeah, I think this whole module needs a thorough documentation pass like we've done here to `matrix.py` and to some extent to `stripe.py`. I had to draw the scope line somewhere just because it's such a big job.